tests/cloud_tests/platforms/lxd/image.py

```diff
@@ -76,19 +76,22 @@ def properties(self):
         }
 
     def export_image(self, output_dir):
-        """Export image from lxd image store to (split) tarball on disk.
+        """Export image from lxd image store to disk
 
-        @param output_dir: dir to store tarballs in
-        @return_value: tuple of path to metadata tarball and rootfs tarball
+        @param output_dir: dir to store the exported image in
+        @return_value: tuple of path to metadata tarball and rootfs
```

Comment:
I'd like the docstring updated with the expected output from lxc image export formats we handle..  Note, that I see at least two formats:

```
71f171df8308fab041101d8d4050312df4ecc1063d73761c5e660e8c983ccd31.squashfs
meta-71f171df8308fab041101d8d4050312df4ecc1063d73761c5e660e8c983ccd31.tar.xz
```

And some images export as a single tarball
```
f9b3b95a1484d7d52132ad342fbf20dc0af1abb457c3ea6594ea58e6fe58d795.tar.gz
```

Reply:
@stgraber makes me notice that LXD just returns what it has been fed and can't change the image format, otherwise the hashes won't match anymore

Reply:
TIL that:

1. There's no way to know in advance the format of an image. It is only possible to know if it's split vs. combined (single .tar.gz) by looking at the number of files in `/var/snap/lxd/common/lxd/images`.
1. The `ubuntu:` and `images:` remotes are simplestreams so the images are always split (no single .tar.gz). The metadata file has always been .tar.xz. The root filesystem can be squashfs, tar.xz, tar.gz (in order or preference). We don't need to really care about the root fs as we're not touching it.

So the current assumptions (split image + .tar.xz metadata) could be good enough for our use case. WDYT @raharper ?

Reply:
That's all rather unfortunate.  In the tar.gz case, it includes the rootfs dir and a metadata.yaml; so there's not metadata tar.xz;


```
 % tar tzvf f9b3b95a1484d7d52132ad342fbf20dc0af1abb457c3ea6594ea58e6fe58d795.tar.gz  | head 
-rw-r--r-- root/root      1048 2020-05-26 04:22 metadata.yaml
drwxr-xr-x 0/0               0 2020-05-26 03:24 rootfs
drwxr-xr-x 0/0               0 2020-05-26 03:10 rootfs/bin
-rwxr-xr-x 0/0         1113504 2019-06-06 17:28 rootfs/bin/bash
```

In the tar.gz case, how are we to return a metadata tarball which there is none?

Reply:
@raharper that's not straightforward I think, but the question is: do we need to cover the single .tar.gz case in our cloud_tests use case, given that by pulling from images: we're always getting split images?